packages/backend/src/route.ts

```diff
@@ -26,22 +11,22 @@ const routes = () => {
   router.get('/', indexController)
 
   // Message route
-  router.get('/message', getAllMessagesController)
-  router.post('/message', createMessageController)
-  router.put('/message', updateMessageController)
-  router.delete('/message', deleteMessageController)
+  //router.get('/message', getAllMessagesController)
+  router.post('/create/message', createMessageController)
+  // router.put('/message', updateMessageController)
+  // router.delete('/message', deleteMessageController)
 
   // Video routes here
-  router.get('/video', getAllVideoController)
-  router.post('/video', createVideoController)
-  router.put('/video', updateVideoController)
-  router.delete('/video', deleteVideoController)
+  //router.get('/video', getAllVideoController)
+  router.post('/create/video', createVideoController)
+  // router.put('/video', updateVideoController)
+  // router.delete('/video', deleteVideoController)
 
   // Fanart route
-  router.get('/fanart', getAllFanartController)
-  router.post('/fanart', createFanartController)
-  router.put('/fanart', updateFanartController)
-  router.delete('/fanart', deleteFanartController)
+  //router.get('/fanart', getAllFanartController)
+  router.post('/create/fanart', createFanartController)
+  // router.put('/fanart', updateFanartController)
+  // router.delete('/fanart', deleteFanartController)
```

Comment:
May I know what's the problems in these APIs? For future improvement

And if we need to do this, if possible I would like to suggest that we implement development branch, so that FE can still work with these API. Like for example I'm currently working on `/fanart` page, and I want to implement the get API if possible

Reply:
Put and delete need some sort of authorization system, I just cut all the gets because at least the message one was in flux and we don't need them yet. I didn't really evaluate them much.

This PR is not going to main and main is still suitable for development.

Reply:
Ok got it